build.proj

```diff
@@ -81,7 +81,12 @@
 
     <Message Importance="High" Text="[$([System.DateTime]::Now.ToString('HH:mm:ss.ff'))] Restoring all packages..." />
 
+    <IsRestoreRequired ProjectJsons="@(ProjectJsonFiles)" PackagesFolder="$(PackagesDir)">
+      <Output TaskParameter="RestoreRequired" PropertyName="RestoreRequired" />
+    </IsRestoreRequired>
+    
     <Exec Command="$(DnuRestoreCommand) @(DnuRestoreDir->'&quot;%(Identity)&quot;', ' ')"
```

Comment:
The name spans a few repos and is used in TFS (last I checked) so it's not a simple rename. It should be renamed, though, and in my opinion to something more generic. (`RestoreCommand`? `ProjectRestoreCommand`?)